src/tx-utils.js

```diff
@@ -392,13 +392,13 @@ async function signBjjAuth(wallet, bjj, chainID, ethAddr) {
     const types = {
         Authorise: [
```

Comment:
I was checking the code of the last `master` commit and saw this typos and others, which are fixed at: https://github.com/hermeznetwork/commonjs/pull/16
This affects the data that is signed, affecting the signature results, so affects also the compatibility with the Go implementation.